criu/include/magic.h

```diff
@@ -99,6 +99,7 @@
 #define PIDNS_MAGIC		0x61157326 /* Surgut */
 #define BPFMAP_FILE_MAGIC	0x57506142 /* Alapayevsk */
 #define BPFMAP_DATA_MAGIC	0x64324033 /* Arkhangelsk */
+#define APPARMOR_MAGIC		0x59423047 /* Sablino */
```

Comment:
Not sure if the coordinates are right =) If I google 59°42'00.0"N 30°47'00.0"E I get to Nikolskoe not Sablino =) The closest "Sablino" I see  is at 59.64216480303743, 30.76562251056352 but yet it was renamed =) "Until 1922, the name of Ulyanovka was Sablino." https://en.wikipedia.org/wiki/Ulyanovka,_Leningrad_Oblast